### PR TITLE
Basheerk testcase time

### PR DESCRIPTION
**Pull request by lmr**
The  time utility runs the specified program/command with the given arguments.
When program/command finishes, time writes a message to standard error giving timing statistics about the program/command run

It provides /usr/bin/time binary

signedoff-By:Basheer K<basheer@linux.vnet.ibm.com>

### CODE REVIEW

**Comment by lmr** [comment]
@basheerk, here is an updated version of your Pull Request. I have fixed some typos I found while reading through the code. There are some other things I don't like about your code:

- Use of carriage return `\n` characters at the beginning of every exception message and logged message (what is up with that)?
- Use of the `commands` library or `os.system`, when autotest does have `utils.run` and `utils.system` utilities that were developed precisely for more convenient execution of subcommands.
- Entirely non intuitive logic to test simple things, such as the code of `test_exit_status_of_cmd`

You are welcome to pull from my branch and create a new Pull Request. @athira-rajeev, please help @basheerk with getting the remarks fixed.

File: linux-tools/test_time/test_time.py [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+
+    def test_exit_status_of_cmd(self, cmd, expected_status):
+        """
+        This method will verify the exit status of the command invoked
+        using time utility.
+        Ex:/usr/bin/time -f "%x is Exit status of the cmd %C" <cmd>
+        """
+        cmd_to_execute = "%s " % self.time_exe + \
+            " -f '%x is the Exit status of the cmd %C' " + cmd
+        search_pattren = "%s is the Exit status of the cmd %s" % (
```
Comment (lmr) [inline linux-tools/test_time/test_time.py]: Oh, `search_pattren`, another typo I didn't catch on my first read...

**Comment by lmr** [comment]
I guess you can strike out my 3rd comment there. You should have been more explicit in the docstring saying that you are testing the `-f` option for `time`.

**Comment by basheerk** [comment]
closing per #71

**Comment by athira-rajeev** [comment]
Hi @lmr    , Thanks for checking. The mentioned changes were added and an updated version of this testcase is merged in PR #71 .
I am closing this PR.

**Pull request closed without merging**